staging/src/k8s.io/kube-aggregator/pkg/controllers/openapi/controller.go

```diff
@@ -102,7 +102,7 @@ func (c *AggregationController) processNextWorkItem() bool {
 		return false
 	}
 
-	klog.Infof("OpenAPI AggregationController: Processing item %s", key)
+	klog.V(5).Infof("OpenAPI AggregationController: Processing item %s", key)
```

Comment:
this seems reasonable to move out of Infof, but the below ones i am not sure

Reply:
It should.The checking interval is one minute default.And log `action for item v1beta1.metrics.k8s.io: Requeue.` just means your openapi spec seems good and wait for the next tick.

Reply:
even though I set log level=0, this log message always printed every second.